config/routes.rb

```diff
@@ -29,5 +29,6 @@
     resources :dashboard, only: [:show]
   end
 
-  resources :stores, only: [:new, :create, :index, :edit]
+  resources :stores, only: [:new, :create, :index, :edit, :update]
+  get 'stores/:id/validate', to: 'stores#validate', as: "validate"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.